README.md

```diff
@@ -177,6 +177,50 @@ xeeApi.getTrips(carId, beginDate, endDate)
 
 All entities from the SDK implement the `Parcelable` interface, making easier the way to pass data between your components
 
+## Sign-In button
+
+Use the Sign-In button to sign in with Xee. Three themes and three sizes are provided 
+
+- **size:** mini, normal, large
+- **theme:** grey, green, white
+
+
+![xee auth buttons](./sign_in_buttons.png)
+
+Use the Sign-In button in layout file
+
+```xml
+<com.xee.auth.SignInButton
+    android:id="@+id/sign_in_button"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    app:signInBtnSize="normal"
+    app:signInBtnTheme="grey"
+    />
+```
+
+You need to set a callback for result when clicking on the button by using the `setOnSignInClickResult` method and pass the `Xee API` instance that you have set (see [Setup](#setup) step)
+
+```java
+signInButton.setOnSignInClickResult(xeeApi, this);
+```
+
+Then implement the `ConnectionCallback` interface and override the `OnError` and `OnSuccess` methods to handle result
+
+```java
+public class MainActivity extends AppCompatActivity implements ConnectionCallback {
```

Comment:
Either you do that, or you can pass an anonymous implementation of `ConnectionCallback` in the method